lib/backend/k8s/k8s_fv_test.go

```diff
@@ -312,4 +318,31 @@ var _ = Describe("Test Syncer API for Kubernetes backend", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(len(objs)).To(Equal(0))
 	})
+
+	It("should support setting and getting GlobalConfig", func() {
+		gc := &model.KVPair{
+			Key: model.GlobalConfigKey{
+				Name: "ClusterGUID",
+			},
+			Value: "someguid",
+		}
+		updGC, err := c.Create(gc)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(updGC.Value.(string)).To(Equal("someguid"))
+		Expect(updGC.Key.(model.GlobalConfigKey).Name).To(Equal("ClusterGUID"))
+		Expect(updGC.Revision).NotTo(BeNil())
+
+		updGC, err = c.Get(gc.Key)
```

Comment:
I assume apply/update should work - in which case we should just add these additional tests....